ddd-koans/Value Object & Entity/Draft/src/Main.kt

```diff
@@ -0,0 +1,21 @@
+import NoteType.*
+import Pitch.*
+
+fun main() {
+    val notes = Notes(
+        listOf(
+            WholeNote.toNote("F2"),
+            QuarterNote.toNote("A1"),
+            EighthNote.toNote("H1"),
+            ThirtySecondNote.toNote("D5"),
+            WholeNote.toNote(F2),
```

Comment:
`WholeNote.toNote` sounds bad to my because `note` part is repeated, what about using constructor-like static factory from `NoteValue` to produce `Note` like `WholeNote("F2") -> Note` or `wholeNote(F2)` or completely resigned from the creation of notes on notes value level and only keep on the Pitch

Reply:
I'll commit version `WholeNote("F2") -> Note`, check whether you like it or not and we can discuss it afterward.